Project1/styles.css

```
@@ -0,0 +1,51 @@
+img {
+    max-width: 100%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+figcaption {
+    text-indent: 5%;
+    font-size: 13px;
+    font-family: helvetica, sans-serif;
```

You ask for Helvetica, but it is not available on all systems. I recommend using a more detailed font stack, e.g.:
```suggestion
    font-family: Helvetica, Arial, "Lucida Grande", sans-serif;
```
or using a sans-serif font available via web fonts (which Helvetica is, but it costs $$$ to use vs Google Fonts).